Content.Client/GameObjects/Components/Mobs/Actions/ActionAssignments.cs

```diff
@@ -109,16 +111,28 @@ public ActionAssignments(byte numHotbars, byte numSlots)
                         continue;
                     }
                 }
+
                 assignmentsWithoutItem.Add(assignmentEntry);
             }
+
             // reassign without the item for each assignment we found that no longer has an associated item
             foreach (var (assignment, slots) in assignmentsWithoutItem)
             {
                 foreach (var (hotbar, slot) in slots)
                 {
-                    if (!assignment.TryGetItemActionWithItem(out var actionType, out _)) continue;
-                    AssignSlot(hotbar, slot,
-                        ActionAssignment.For(actionType));
+                    if (!assignment.TryGetItemActionWithItem(out var actionType, out _))
+                    {
+                        continue;
+                    }
+
+                    if (actionMenuLocked)
```

Comment:
This is all from your other PR isn't it? Might want to remove these changes.

Reply:
this is because I based this branch on that other branch, because it had some .ftl changes also. I think it would be best of the other PR is merged first (I put this info about this in this PR's description + already updated the other PR with all suggested changes, so it should be fine)